cvat/apps/engine/task.py

```diff
@@ -299,7 +345,6 @@ def _validate_manifest(manifests, root_dir, is_in_cloud, db_cloud_storage, data_
                 raise ValidationError("Manifest file can be uploaded only if 'Use cache' option is also selected")
             return manifest_file
         raise ValidationError('Invalid manifest was uploaded')
-    return None
```

Comment:
In python, every function returns something. If it doesn't have a return statement, then it returns None. This matches the declared type.

Reply:
I know, but from a maintainability standpoint, if a function is supposed to return a value, then it should end with a return statement, otherwise it looks like you just forgot to return something.

https://peps.python.org/pep-0008/#programming-recommendations